composer.json

```diff
@@ -38,6 +38,7 @@
     "cspray/phinal": "^2.0.0",
     "doctrine/orm": "^2.18.0|^3.0.0",
     "infection/infection": "^0.27.0",
+    "league/commonmark": "^2.4",
```

Comment:
maybe move to tools?

Reply:
You could try it out and see if it works. But I don't want to do that here anymore. I had to start over 3 times because I had an error in the script. I want to save the status for now :D